angular/projects/spark-angular/src/lib/directives/inputs/sprk-helper-text/sprk-helper-text.directive.spec.ts

```diff
@@ -4,7 +4,11 @@ import { SprkHelperTextDirective } from './sprk-helper-text.directive';
 
 @Component({
   selector: 'sprk-test',
-  template: ` <span sprkHelperText>Helper Text</span> `,
+  template: `
+    <span sprkHelperText idString="test-str" analyticsString="test"
+      >Helper Text</span
+    >
```

Comment:
Why did prettier (I am assuming it was prettier) put the closing span tag on an entirely new line in this way.

Reply:
That's an excellent question, but I hate the way it looks. I'll see if it lets me change it.